assets/styles/elements/Header/_HeaderNavigationSubMenu.scss

```diff
@@ -407,4 +430,28 @@
 		}
 	}
 
+	@media (min-width: $breakpoint-desktop) {
```

Comment:
Would try to simplify and reduce nesting if possible (but due to menu of WP might be not)

Reply:
That is not possible.